DigitalLearningSolutions.Data/DataServices/CategoriesDataService.cs

```diff
@@ -0,0 +1,33 @@
+﻿namespace DigitalLearningSolutions.Data.DataServices
+{
+    using System.Collections.Generic;
+    using System.Data;
+    using Dapper;
+    using DigitalLearningSolutions.Data.Models.Common;
+
+    public interface ICategoriesDataService
+    {
+        IEnumerable<Category> GetCategoryListForCentre(int centreId);
+    }
+
+    public class CategoriesDataService : ICategoriesDataService
+    {
+        private readonly IDbConnection connection;
+
+        public CategoriesDataService(IDbConnection connection)
+        {
+            this.connection = connection;
+        }
+
+        public IEnumerable<Category> GetCategoryListForCentre(int centreId)

```

Comment:
Could we name this GetCategoriesForCentreAndCentrallyManagedCourses? It's a bit of a mouthful, but I think it'll self-document the "CentreId = 0" part :)